### PR TITLE
Add `const` var declaration to FileSystemFileHandle sample code 

### PR DESCRIPTION
**Pull request by ZhangYiJiang**
#### Summary

Add `const` variable declaration in sample code

#### Motivation

This prevents the variable from leaking from the function scope 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/filesystemfilehandle/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ const pickerOpts = {
 
 async function getTheFile() {
   // open file picker
-  [fileHandle] = await window.showOpenFilePicker(pickerOpts);
+  const [fileHandle] = await window.showOpenFilePicker(pickerOpts);
 
   // get file contents
   const fileData = await fileHandle.getFile();
```
Comment (Josh-Cena) [inline files/en-us/web/api/filesystemfilehandle/index.md:62]: Should we return this, or at least add a comment below that we are supposed to use this? Currently this is unused.

Comment (teoli2003) [reply files/en-us/web/api/filesystemfilehandle/index.md:62]: Yes, we should return something. What about:
```suggestion
  return await fileHandle.getFile();
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
In addition to the return value, should we put the definition of `pickerOpts` inside the function?

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**